pms/models/pms_reservation.py

```diff
@@ -833,19 +824,21 @@ def name_get(self):
 
     @api.model
     def create(self, vals):
-        if "folio_id" in vals and "channel_type" not in vals:
+        if "folio_id" in vals and "channel_type_id" not in vals:
```

Comment:
You dont need use this, because channel_type_id is related field (the channel_type_id on reservation is always the same as the folio)